core/web/apiv2/observables.py

```diff
@@ -282,36 +280,83 @@ async def add_text(request: AddTextRequest) -> ObservableTypes:
     """Adds and returns an observable for a given string, attempting to guess
     its type."""
     try:
-        return Observable.add_text(request.text, request.tags)
+        return observable.save(value=request.text, tags=request.tags)
     except ValueError as error:
         raise HTTPException(status_code=400, detail=str(error))
 
 
+@router.post("/import/text")
+async def import_from_text(request: ImportTextRequest) -> BulkObservableAddResponse:
```

Comment:
Can we just deprecate `add_text` in this case? They seem to be covering pretty much exaclty the same use-case?

If we're worried about breaking APIs, then I'm fine keeping add_text, but maybe we can just add the `@router.post` handler to this function instead?

(also, does FastAPI handle deprecation notices in endpoints?)

Reply:
fixed in 8417c1187e3db0f0c5d7d86d2eeb1bba47ec6b7d